packages/protocol/contracts/governance/EpochRewards.sol

```diff
@@ -316,8 +316,10 @@ contract EpochRewards is Ownable, Initializable, UsingPrecompiles, UsingRegistry
    */
   function getVotingGoldFraction() public view returns (uint256) {
     // TODO(asa): Ignore custodial accounts.
-    address reserveAddress = registry.getAddressForOrDie(RESERVE_REGISTRY_ID);
-    uint256 liquidGold = getGoldToken().totalSupply().sub(reserveAddress.balance);
+    //address reserveAddress = registry.getAddressForOrDie(RESERVE_REGISTRY_ID);
+    //uint256 liquidGold = getGoldToken().totalSupply().sub(reserveAddress.balance);
+    uint256 reserveBalance = getReserve().getReserveGoldBalance();
+    uint256 liquidGold = getGoldToken().totalSupply().sub(address(getReserve()).balance);
```

Comment:
Can you remove the `TODO` below this one? It should in fact be total votes

Reply:
Ok. What about TODO(asa): Ignore custodial accounts?  Seems this PR stops ignoring custodial accounts.

Reply:
Yeah I think that TODO can be removed as well